GISSR_FINAL CHALLENGE_REFILOE SEKALELI.Rmd

```diff
@@ -172,94 +179,157 @@ scale_fill_manual(name = "Land Use",
+```{r}
+image_url<- ("https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRiLbJ_chzJrCIJ2UmlWeDOyx8Cta09-PQJ7A&s")
+```
 
-Using what you have learned so far in this course, create at least one "static" map and one "interactive" map for your chosen location. Start with just a rough draft. You are welcome to create more than two maps, but start with two and complete steps 5 and 6 for those before you add any more. 
+<img src="`r here::here('Images/ALL-SPICE.png')`" width="240" /> <img src="`r here::here('Images/3_CIFAL.png')`" width="200" />
 
-Don't reinvent the wheel if you don't have to. Use the code we have practiced with to get started, and alter the code for your location and topic. If you have draft maps from our previous exercises that work for this, use those! If you can find code online that is similar to what you want to do, use it!
+```{r}
+bb %>%
+  opq()
+```
 
-### Step 5: Interpret and explain your maps
+```{r}
+hospital <- bb %>%
+opq() %>%
+		add_osm_feature(key = "amenity", value = "hospital") %>%
+		osmdata_sf()
+
+clinic <- bb %>%
+	opq() %>%
+		add_osm_feature(key = "amenity", value = "clinic") %>%
+		osmdata_sf() 
+
+chemist<- bb %>%
+    	opq() %>%
+		add_osm_feature(key = "shop",value =  "chemist") %>%
+		osmdata_sf()
+  
+pharmacy <- bb %>%
+	opq() %>%
+		add_osm_feature(key = "amenity", value = "pharmacy") %>%
+		osmdata_sf()
+
+specialist <- bb %>%
+	opq() %>%
+		add_osm_feature(key = "healthcare", value = "specialist") %>%	
+		osmdata_sf()
+
+counselling<- bb %>%
+	opq() %>%
+		add_osm_feature(key = "healthcare", value = "counselling") %>%
+			osmdata_sf() 
 
-Add text above and below your map that explains what we can see in the data. This is your explanation and interpretation of the data. 
+streets <- bb %>%
+  opq() %>%
+    add_osm_feature("highway", c("primary", "secondary", "tertiary")) %>%
+    osmdata_sf()
 
-This should include:
+```
+```{r}
+hospital
+chemist
+pharmacy
+clinic
+specialist
+counselling
+streets
+```
+##1.1 DATA SOUCE AND OBSERVATION
+The static map below show all healthcare facilities withing the bounding box of Mafeteng and their proximity to villages and roads for ease of access for the sick. The analysis was done using osm data for hospitals, clinics, chemists, pharmacies, specialists ,wholesalers and shops within the area of Mafeteng.
+Only one hospital was identified in the central region of Mafeteng and another hospital outside the Mafeteng boundaries and a total of four health centers were identified. 
+As per BOS Lesotho 2016 statistics Mafeteng has a population of 178,222 and one hospital is expected to service the entire population. Hence the high rate of infant mortality and death rates within the region. 
+```{r}
+ggplot(data = world) +
```

Comment:
can use just ggplot() + to initialize the ggplot (not using the "world" layer for this map) and then just add the geom_sf as you did below